src/deepsparse/tasks.py

```diff
@@ -69,13 +69,22 @@ class SupportedTasks:
     """
 
     nlp = namedtuple(
-        "nlp", ["question_answering", "text_classification", "token_classification"]
+        "nlp",
+        [
+            "question_answering",
+            "text_classification",
+            "token_classification",
+            "embedding_extraction",
+            "haystack",
```

Comment:
let's make the main name `information_retrieval_haystack` with `haystack` as an alias